docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,90 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a main `.bicepparam` file that can be consumed by multiple extended `.bicepparam` files.
+
+## Using
+
+`main.bicep` This is your main bicep file, which will define your parameters for deployment.
+
+```bicep
+param namePrefix string
+param location string
+param symbolicName string
```

Comment:
any other parameters we can use instead of `symbolicName`? Symbolic Name is a specific concept in bicep and it cannot be parameterized, so it is a strange example.

Reply:
How does `tag` sound? I know several users tend to add tags to their deployments.